web/src/app/services/UniversalKey/ExprField.tsx

```diff
@@ -21,13 +24,25 @@ export function ExprField(props: ExprFieldProps): React.ReactNode {
       variant='outlined'
       data-testid={'code-' + props.name}
     >
-      <InputLabel htmlFor={props.name} shrink>
-        {props.label} (Expr syntax)
+      <InputLabel htmlFor={props.name} sx={{ padding: '0 50px 0 0' }}>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          {props.label}
+          <Link
+            href='https://expr-lang.org/docs/language-definition'
+            target='_blank'
+            rel='noreferrer'
+            style={{ marginTop: '5px', marginRight: '5px', marginLeft: '5px' }}
+          >
+            <Tooltip title='Expr syntax'>
+              <InfoOutlinedIcon />
+            </Tooltip>
+          </Link>
```

Comment:
You'll have to add the import, but use `AppLink` for any links for consistency 

```suggestion
          <AppLink
            to='https://expr-lang.org/docs/language-definition'
            newTab
            style={{ marginTop: '5px', marginRight: '5px', marginLeft: '5px' }}
          >
            <Tooltip title='Expr syntax'>
              <InfoOutlinedIcon />
            </Tooltip>
          </Link>
```